### ISSUE TITLE
Explanatory document for newbies

### ISSUE THREAD

**Issue by ddooley** [EnvironmentOntology/gaz#16]
GAZ needs an explanatory document for newbies – how to get a country's 2nd level subdivisions, whether they are states/provinces/territories/regions etc. and the linkage between geopolitical levels of government entities using "located in".  Same for municipalities, villages, ENVO human habitation etc… Unless there is some documentation about this that I don't know about?

**Comment by cmungall** [comment]
Yes! This is something that can be done independent on tackling the bigger
Q of how GAZ should be maintained. I suggest either (1) putting all this in
the main README.md or (2) using the wiki on this GH repo. Either way the
text will be markdown and can easily be transferred.

**Comment by pbuttigieg** [comment]
+1

**Comment by lschriml** [comment]
Good idea !!

**Comment by cmungall** [comment]
This will be dependent on #20 

Any documentation we write on how to do this on the class representation will not be applicable to the instance one
